data_steward/analytics/cdr_ops/controlled_tier_qc/check_controlled_tier_part2.py

```diff
@@ -473,7 +473,14 @@ def my_sql(table_name, column_name):
 WHERE observation_concept_id NOT IN (40766240,43528428,1585389) 
 AND concept_class_id='Module'
 AND concept_name IN ('The Basics') 
-AND src_id='PPI/PM'
+AND src_id IN (
+  SELECT 
+    src_id
+  FROM
+    `{{project_id}}.pipeline_tables.site_maskings`
+  WHERE NOT
+    REGEXP_CONTAINS(src_id, r'(?i)(PPI/PM)|(EHR site)')
+)
```

Comment:
Nit: This addition does follow the original meaning but it might be simplified with AND NOT REGEXP_CONTAINS(src_id, r'(?i)(PPI/PM)|(EHR site)') in line 476. 

I don't know if the replacement will completely remove the use of 'PPI/PM'. Depending on that the statement could look like this AND NOT REGEXP_CONTAINS(src_id, r'(?i)EHR site').